draft-irtf-cfrg-hpke.md

```diff
@@ -177,19 +205,22 @@ for any combination of an asymmetric key encapsulation mechanism
 (KEM), key derivation function (KDF), and authenticated encryption
 with additional data (AEAD) encryption function. We provide
 instantiations of the scheme using widely-used and efficient
-primitives.
+primitives based on elliptic curve Diffie-Hellman key agreement.
```

Comment:
Isn't HPKE supposed to be more general than this? SIKE should be included as a possible KEM, for example.

Reply:
It is, but this sentence pertains to what’s currently specified (with a code point) in the document.